stdlib/LinearAlgebra/src/tridiag.jl

```diff
@@ -693,6 +697,10 @@ end
     end
 end
 
+Base._reverse(A::Tridiagonal, dims) = reverse!(Matrix(A); dims)
+Base._reverse(A::Tridiagonal, dims::Colon) = Tridiagonal(reverse(A.dl), reverse(A.d), reverse(A.du))
```

Comment:
```suggestion
Base._reverse(A::Tridiagonal, dims::Colon) = Tridiagonal(reverse(A.du), reverse(A.d), reverse(A.dl))
```
Reversal swaps the upper and lower diagonals in addition to reversing their order.